packages/terra-application-links/tests/wdio/application-tabs-spec.js

```diff
@@ -1,118 +1,128 @@
 const viewports = Terra.viewports('small', 'large');
 
 describe('ApplicationTabs', () => {
-  describe('Displays a default application tabs', () => {
-    beforeEach(() => {
-      browser.url('/#/raw/tests/terra-application-links/application-links/application-tabs-default');
-      browser.waitForVisible('#test-tabs [data-application-tabs-more]');
-      browser.moveToObject('#test-tabs [data-application-tabs-more]');
+  viewports.forEach((viewport) => {
```

Comment:
You might need this loop outside the describe. I am seeing this screenshot different for the large viewport size

![screen shot 2019-02-15 at 12 09 05 pm](https://user-images.githubusercontent.com/14099737/52875755-bb222a80-311a-11e9-8469-fe9219abac47.png)

Reply:
Updated good catch, that fixed it.